google/ggrc-core
# GGRC-6656 Display Last Updated Date of snapshot in tree-view

# Issue description

Incorrect sorting of Last Updated date in snapshot tree-view.

# Steps to test the changes

Steps to reproduce:
1. Log in GGRC as admin user
2. Open any audit page and map at least 10 controls
3. In control tab look at Last Updated date of control snapshots

Actual Result: Incorrect sorting of Last Updated date of control snapshots 
Expected Result: Sorting of Last Updated date of control snapshots is correct

# Solution description

We use `toObject` snapshot util to convert content of revision to displayable object. So previously we displayed last updated date of original object.
Solution is manually set `updated_at` to object from snapshot.

# Sanity checklist




# PR Review checklist
